models/ldndc/R/met2model.LDNDC.R

```diff
@@ -0,0 +1,247 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2012 University of Illinois, NCSA.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the 
+# University of Illinois/NCSA Open Source License
+# which accompanies this distribution, and is available at
+# http://opensource.ncsa.illinois.edu/license.html
+#-------------------------------------------------------------------------------
+
+##-------------------------------------------------------------------------------------------------#
+##' Converts a met CF file to a model specific met file. The input
+##' files are calld <in.path>/<in.prefix>.YYYY.cf
+##'
+##' @name met2model.LDNDC
+##' @title Write LDNDC met files
+##' @param in.path path on disk where CF file lives
+##' @param in.prefix prefix for each file
+##' @param outfolder location where model specific output is written.
+##' @param overwrite logical: replace output files if they already exist?
+##' @return OK if everything was successful.
+##' @export
+##' @author Henri Kajasilta
+##-------------------------------------------------------------------------------------------------#
+met2model.LDNDC <- function(in.path, in.prefix, outfolder, start_date, end_date, overwrite = FALSE, ...) {
+  
+  # Logger info
+  PEcAn.logger::logger.info("START met2model.LDNDC")
+  
+  # Years
+  start_year <- lubridate::year(start_date)
+  end_year <- lubridate::year(end_date)
+  
+  
+  # Check the nc-files
+  nc_file <- list.files(in.path)[grep("*.nc", list.files(in.path))]
+  
+  # Set-up the outfile that will be returned as results
+  out.file <- paste(in.prefix, format(as.Date(start_date), "%Y-%m-%d"),
+                    format(as.Date(end_date), "%Y-%m-%d"),
+                    "txt",
+                    sep = ".")
+  
+  out.file.full <- file.path(outfolder, out.file)
+  
+  # Results
+  results <- data.frame(file = out.file.full,
+                        host = PEcAn.remote::fqdn(),
+                        mimetype = "text/plain",
+                        formatname = "LDNDC_Climate",
+                        startdate = start_date,
+                        enddate = end_date,
+                        dbfile.name = out.file,
+                        stringsAsFactors = FALSE)
+  
+  
+  
+  if(length(nc_file) > 0){
+    print("At least one nc-file was found from the given path")
```

Comment:
I believe we are using `logger` functions instead of `print`. Was this line a remnant of debugging/developing? If it was meant to be informative, could you revise the comment a little bit? E.g. on the line of what it means if there is at least one nc-file for the requested run. Or is it better to check for `length(nc_file) == 0` condition and fail here?

Reply:
Changed to `length(nc_file) == 0` and logger.severe